CRITICAL - 2024-06-23 05:51:13 --> ErrorException: Undefined array key "rol"
in APPPATH\Views\maestros\edit.php on line 53.
 1 APPPATH\Views\maestros\edit.php(53): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "rol"', 'C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\maestros\\edit.php', 53)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\maestros\\edit.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('maestros/edit', [], true)
 5 APPPATH\Controllers\Maestros.php(82): view('maestros/edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Maestros->edit('18')
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Maestros))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 05:51:38 --> ErrorException: Undefined array key "rol"
in APPPATH\Views\maestros\edit.php on line 53.
 1 APPPATH\Views\maestros\edit.php(53): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "rol"', 'C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\maestros\\edit.php', 53)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\maestros\\edit.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('maestros/edit', [], true)
 5 APPPATH\Controllers\Maestros.php(82): view('maestros/edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Maestros->edit('16')
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Maestros))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-23 07:45:13 --> mysqli_sql_exception: Unknown column 'idPadre' in 'where clause' in C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\prueba\SistemaControl\app\Controllers\Padres.php(153): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('1')
#8 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
#9 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\prueba\SistemaControl\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-06-23 07:45:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'idPadre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = :idPadre:', [...], false)
 2 SYSTEMPATH\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Padres.php(153): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('1')
 6 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 07:45:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'idPadre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = :idPadre:', [...], false)
 4 SYSTEMPATH\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Padres.php(153): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('1')
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 07:45:13 --> [Caused by] mysqli_sql_exception: Unknown column 'idPadre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = :idPadre:', [...], false)
 5 SYSTEMPATH\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Padres.php(153): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('1')
 9 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-23 07:45:23 --> mysqli_sql_exception: Unknown column 'idPadre' in 'where clause' in C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\prueba\SistemaControl\app\Controllers\Padres.php(153): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('3')
#8 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
#9 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\prueba\SistemaControl\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-06-23 07:45:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'idPadre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = :idPadre:', [...], false)
 2 SYSTEMPATH\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Padres.php(153): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('3')
 6 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 07:45:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'idPadre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = :idPadre:', [...], false)
 4 SYSTEMPATH\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Padres.php(153): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('3')
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 07:45:23 --> [Caused by] mysqli_sql_exception: Unknown column 'idPadre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = :idPadre:', [...], false)
 5 SYSTEMPATH\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Padres.php(153): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('3')
 9 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-23 07:52:00 --> mysqli_sql_exception: Unknown column 'idPadre' in 'where clause' in C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\prueba\SistemaControl\app\Controllers\Padres.php(131): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('1')
#8 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
#9 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\prueba\SistemaControl\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-06-23 07:52:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'idPadre' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = :idPadre:', [...], false)
 2 SYSTEMPATH\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Padres.php(131): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('1')
 6 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 07:52:00 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'idPadre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = :idPadre:', [...], false)
 4 SYSTEMPATH\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Padres.php(131): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('1')
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 07:52:00 --> [Caused by] mysqli_sql_exception: Unknown column 'idPadre' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = \'1\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `datos_alumnos`
WHERE `idPadre` = :idPadre:', [...], false)
 5 SYSTEMPATH\Model.php(243): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(641): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Padres.php(131): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->getAlumnosAjax('1')
 9 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 08:02:55 --> ErrorException: Undefined array key "genero"
in APPPATH\Views\padres\edit.php on line 28.
 1 APPPATH\Views\padres\edit.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "genero"', 'C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\padres\\edit.php', 28)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\padres\\edit.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('padres/edit', [], true)
 5 APPPATH\Controllers\Padres.php(85): view('padres/edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 08:03:09 --> ErrorException: Undefined array key "genero"
in APPPATH\Views\padres\edit.php on line 28.
 1 APPPATH\Views\padres\edit.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "genero"', 'C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\padres\\edit.php', 28)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\padres\\edit.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('padres/edit', [], true)
 5 APPPATH\Controllers\Padres.php(85): view('padres/edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 08:21:19 --> ErrorException: Undefined property: CodeIgniter\View\View::$endSection
in APPPATH\Views\padres\edit.php on line 168.
 1 APPPATH\Views\padres\edit.php(168): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: CodeIgniter\\View\\View::$endSection', 'C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\padres\\edit.php', 168)
 2 SYSTEMPATH\View\View.php(228): include('C:\\laragon\\www\\prueba\\SistemaControl\\app\\Views\\padres\\edit.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('padres/edit', [], true)
 5 APPPATH\Controllers\Padres.php(85): view('padres/edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-23 23:01:00 --> mysqli_sql_exception: Table 'sistema_control.alumnos' doesn't exist in C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\prueba\SistemaControl\app\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\prueba\SistemaControl\app\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
#6 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
#7 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
#8 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\prueba\SistemaControl\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-06-23 23:01:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'sistema_control.alumnos' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = :responsable_alumno.idDatosResponsable:', [...], false)
 2 APPPATH\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 23:01:00 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'sistema_control.alumnos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = :responsable_alumno.idDatosResponsable:', [...], false)
 4 APPPATH\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 23:01:00 --> [Caused by] mysqli_sql_exception: Table 'sistema_control.alumnos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = :responsable_alumno.idDatosResponsable:', [...], false)
 5 APPPATH\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-23 23:01:50 --> mysqli_sql_exception: Table 'sistema_control.alumnos' doesn't exist in C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\prueba\SistemaControl\app\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\prueba\SistemaControl\app\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
#6 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
#7 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
#8 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\prueba\SistemaControl\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-06-23 23:01:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'sistema_control.alumnos' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = :responsable_alumno.idDatosResponsable:', [...], false)
 2 APPPATH\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 23:01:50 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'sistema_control.alumnos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = :responsable_alumno.idDatosResponsable:', [...], false)
 4 APPPATH\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 23:01:50 --> [Caused by] mysqli_sql_exception: Table 'sistema_control.alumnos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = :responsable_alumno.idDatosResponsable:', [...], false)
 5 APPPATH\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-23 23:01:53 --> mysqli_sql_exception: Table 'sistema_control.alumnos' doesn't exist in C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\prueba\SistemaControl\app\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\prueba\SistemaControl\app\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
#6 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
#7 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
#8 C:\laragon\www\prueba\SistemaControl\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\prueba\SistemaControl\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-06-23 23:01:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'sistema_control.alumnos' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = :responsable_alumno.idDatosResponsable:', [...], false)
 2 APPPATH\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 23:01:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'sistema_control.alumnos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = :responsable_alumno.idDatosResponsable:', [...], false)
 4 APPPATH\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
 6 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 7 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 23:01:53 --> [Caused by] mysqli_sql_exception: Table 'sistema_control.alumnos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = \'4\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `responsable_alumno`
JOIN `alumnos` ON `alumnos`.`idAlumno` = `responsable_alumno`.`idAlumno`
WHERE `responsable_alumno`.`idDatosResponsable` = :responsable_alumno.idDatosResponsable:', [...], false)
 5 APPPATH\Models\ResponsableAlumnoModel.php(18): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Padres.php(84): App\Models\ResponsableAlumnoModel->getAlumnosAsociados('4')
 7 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->edit('4')
 8 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 23:27:34 --> ErrorException: Undefined array key 1
in APPPATH\Controllers\Padres.php on line 112.
 1 APPPATH\Controllers\Padres.php(112): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 1', 'C:\\laragon\\www\\prueba\\SistemaControl\\app\\Controllers\\Padres.php', 112)
 2 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->update('1')
 3 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-23 23:28:27 --> ErrorException: Undefined array key 1
in APPPATH\Controllers\Padres.php on line 112.
 1 APPPATH\Controllers\Padres.php(112): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 1', 'C:\\laragon\\www\\prueba\\SistemaControl\\app\\Controllers\\Padres.php', 112)
 2 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Padres->update('1')
 3 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Padres))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
